<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>799</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CyberPodium</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="centralWidget" native="true">
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">
        QWidget#left_wdg{
            background-color: rgb(5,186,251); 
        }
        </string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,3">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="left_wdg" native="true">
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEditNickname"/>
         </item>
         <item row="6" column="1">
          <widget class="QLineEdit" name="lineEditOffice"/>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="lineEditWeChat"/>
         </item>
         <item row="5" column="1">
          <widget class="QLineEdit" name="lineEditPhone"/>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="labelOffice">
           <property name="text">
            <string>办公室:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="labelEmail">
           <property name="text">
            <string>邮箱:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="labelWeChat">
           <property name="text">
            <string>微信:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="lineEditAccount"/>
         </item>
         <item row="7" column="0" colspan="2">
          <widget class="QTextEdit" name="textEditRemarks">
           <property name="placeholderText">
            <string>备注</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="labelNickname">
           <property name="text">
            <string>昵称:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="lineEditEmail"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="labelAccount">
           <property name="text">
            <string>账号:</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="labelPhone">
           <property name="text">
            <string>电话:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="labelQQ">
           <property name="text">
            <string>QQ:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="lineEditQQ"/>
         </item>
         <item row="8" column="0">
          <widget class="QPushButton" name="EditMsgBtn">
           <property name="text">
            <string>修改用户信息</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="right_wdg" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_2" stretch="5,1">
         <property name="leftMargin">
          <number>5</number>
         </property>
         <property name="topMargin">
          <number>5</number>
         </property>
         <property name="rightMargin">
          <number>5</number>
         </property>
         <property name="bottomMargin">
          <number>5</number>
         </property>
         <item>
          <widget class="QWidget" name="course_wdg" native="true">
           <property name="autoFillBackground">
            <bool>true</bool>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QListWidget" name="courselist_wdg"/>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QTextBrowser" name="select_view">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Ignored">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="lesson_btn">
              <property name="text">
               <string>开始上课</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="enter_btn">
              <property name="text">
               <string>进入课程</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="new_btn">
              <property name="text">
               <string>新建课程</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="delete_btn">
              <property name="text">
               <string>删除课程</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
